src/jdk.hotspot.agent/linux/native/libsaproc/ps_core.c

```diff
@@ -569,8 +569,13 @@ static uintptr_t calc_prelinked_load_address(struct ps_prochandle* ph, int lib_f
 
   free(phbuf);
 
+#if defined(__clang__)
+if (ps_pdread(ph, (char *)link_map_addr + LINK_MAP_LD_OFFSET,
+               &lib_ld, sizeof(uintptr_t)) != PS_OK) {
+#else
```

Comment:
What problem is being "fixed" by these?  I'm dubious that this is the best solution to whatever the problem
is, but can't evaluate or suggest alternatives without knowing what it is.

Reply:
done